bin/ddg2logo

```diff
@@ -49,7 +44,7 @@ accepted (or not accepted) mutations, the respective column can be replaced by a
 controlled by the -x option. Before plotting the following transformations are
 carried out on the data:
 
-STABILIZING MUTATIONS    
+STABILIZING MUTATIONS
 * the DDG values >= than the stabilizing threshold (option -T) are set equal to 0 
```

Comment:
I think you should remove the "="

Reply:
I changed it in the code because I think it is ok to keep the 0 as a possible value. Now it should work when providing '-t 0' or '-T 0' too